charts/telemetry/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+name: telemetry
+description: A Helm chart to install agents
+version: v0.1.0
+keywords:
+- telemetry
+maintainers:
+- name: Jiacheng Xu
+  email: support@kubermatic.com
+- name: Harshita Sharma
+  email: support@kubermatic.com
+
+
+
+
```

Comment:
Please trim your trailing newlines to be exactly 1.

Please also define an `appVersion` field.

Please do not name yourselves as maintainers, use the same maintainer we use for all other Helm charts.

Reply:
@xrstf Where/How do we use this `appVersion` in KKP?